spec-dtslint/observables/combineLatest-spec.ts

```diff
@@ -123,3 +123,13 @@ it('should accept 6 params and a result selector', () => {
 it('should accept 7 or more params and a result selector', () => {
   const o = combineLatest([a$, b$, c$, d$, e$, f$, g$, g$, g$], (a: any, b: any, c: any, d: any, e: any, f: any, g1: any, g2: any, g3: any) => new A()); // $ExpectType Observable<A>
 });
+
+describe('combineLatest({})', () => {
+  it('should properly type empty objects', () => {
+    const res = combineLatest({}); // $ExpectType Observable<never>
+  });
+
+  it('should work for the simple case', () => {
+    const res = combineLatest({ foo: of(1), bar: of('two'), baz: of(false) }); // $ExpectType Observable<{ foo: number; bar: string; baz: boolean; }>
```

Comment:
I would prefer this test to use the helpers - `a$`, etc. - instead of `of` with literal values.

I can see that what's here is based on the `forkJoin` tests - which use `of` and literal values - but that's because those tests have not been refactored to use the helpers. Going forward, we should favour using the helpers.

Reply:
I see the helpers are - so far - only used in `spec-dtslint/operators/*.spec.ts` and not in `spec/observables/*.spec.ts`.
Can I just import those same helpers in `spec/`? (as they are in `spec-dtslint/helpers`)

Reply:
Nope. They're only for use - at least ATM - in the dtslint tests. They offer a somewhat more disciplined approach regarding the types - there are only a few different literal types, but we can add as many distinct 'helper' types as is necessary. With the tests in `spec`, we're less interested in the types and more interested in the notifications and their values and strings, etc. would be fine there.